website/templates/project/statistics.mako

```diff
@@ -5,6 +5,16 @@
   <h2 class="text-300">Analytics</h2>
 </div>
 
+<div id="hiddenFlag" class="ad_block_display">
+  <div class="m-b-md p-md osf-box-lt box-round text-center">The use of adblocking software may prevent site analytics from loading properly.</div>
+</div>
+<script src="/static/js/ads.js"></script>
+<script>
+  if( window.canRunAds === undefined ){
+    document.getElementById('hiddenFlag').style.display = "block";
```

Comment:
If you replace out https://github.com/CenterForOpenScience/osf.io/pull/5678/files#diff-8b5fde9e5d5d19f47a0415fa762de65bR6 with this line, and instead of `document.getElementById('hiddenFlag')` you use `$.('.ad_block_display)`, then you can entirely remove this script block. That means that anything that has the class `ad_block_display` and that imports the ads.js script will do what you want it to do. Nice and reusable.

Are you familiar with jQuery? If not, take a look at it before implementing the change. It's super-handy and great for maintaining cross-browser compatibility.